cmake/scripts/linux/Install.cmake

```diff
@@ -92,6 +96,11 @@ install(FILES ${CMAKE_BINARY_DIR}/${CORE_BUILD_DIR}/${APP_NAME_LC}.desktop
         DESTINATION ${datarootdir}/applications
         COMPONENT kodi)
 
+# Install metainfo
+install(FILES ${CMAKE_BINARY_DIR}/${CORE_BUILD_DIR}/${APP_NAME_LC}.metainfo.xml
+        DESTINATION app/share/metainfo
```

Comment:
absolute path please

Reply:
That actually was the wrong path too. Should be better now.

Reply:
sorry, I guess I wasn't clear. We need to use 
${datarootdir}/metainfo
or it will break installing to alternate paths

Reply:
Should be fixed now. I have no idea if that get's us to the correct folder, but we'll see :)